SOURCES/anicorn

```diff
@@ -317,6 +330,47 @@ isUnicornRestarted() {
   return 1
 }
 
+# Run anicorn as process monitor current Unicorn process
+#
+# 1: PID (Number)
+#
+# Code: No
+# Echo: No
+runWatchdog() {
+  local pid
+
+  pid=$(getPID)
+
+  if [[ -z "$pid" ]] ; then
+    log "[ERROR] Can't start watchdog: can't find Unicorn PID"
+    return
+  fi
+
+  $BIN --watchdog "$pid" &>/dev/null &
+
+  disown
+}
+
+# Terminate Unicorn if it isn't restarted
+#
+# 1: PID (Number)
+#
+# Code: No
+# Echo: No
+monitorStoppedUnicorn() {
+  local pid="$1"
+
+  for i in $(seq 0 $DELAY_PID_MONITOR) ; do
+    if [[ ! -e "/proc/$pid" ]] ; then
+      exit 0
+    fi
+
+    sleep 1
+  done
+
+  kill -TERM "$pid"
```

Comment:
https://bogomips.org/unicorn/SIGNALS.html

* INT/TERM - quick shutdown, kills all workers immediately
* QUIT - graceful shutdown, waits for workers to finish their current request before finishing

I think `-QUIT` is a better option here.

Reply:
I saw this documentation and I select `TERM` signal because after given delay Unicorn **must be** terminated, and `TERM` is 100% way to kill process.

I can change it to `QUIT` if it's isn't quite safe.

Reply:
OK then, let it be TERM.